app/_hub/tomkerkhove/microsoft_azure/index.md

```diff
@@ -0,0 +1,158 @@
+---
+name: Microsoft Azure
+publisher: Tom Kerkhove
+
+categories:
+  - deployment
+
+type: integration
+
+desc: Deploy Kong on Microsoft Azure # (required) 1-liner description; max 80 chars
+description: #|
+  This guide walks you through the options to run Kong on Microsoft Azure.
+  # (required) extended description.
+  # Use YAML piple notation for extended entries.
+  # EXAMPLE long text format (do not use this entry)
+  # description: |
+  #   Maintain an indentation of two (2) spaces after denoting a block with
+  #   YAML pipe notation.
+  #
+  #   Lorem Ipsum is simply dummy text of the printing and typesetting
+  #   industry. Lorem Ipsum has been the industry's standard dummy text ever
+  #   since the 1500s.
+
+support_url: # (Optional) A specific URL of your own for this extension.
```

Comment:
This needs a URL - I'm not clear what it would be - maybe some Bitnami support URL or ??

Reply:
Good point :+1: - Will add one to my repo, can still point them to Bitnami if it's for the solution templates (can grow in future)